src/main/resources/lang/english.lang

```diff
@@ -1242,8 +1273,11 @@ types:
 	damagecause: damage cause¦s @a
 	teleportcause: teleport cause¦s @a
 	inventoryaction: inventory action¦s @a
-	clickaction: click actions¦s @a
+	clicktype: click type¦s @a

```

Comment:
click[ed] type

click type, clicked type

Reply:
Nouns are handled differently and thus they don't allow this.